src/SeaIceDeformation/M01_d03_compute_deformations.py

```diff
@@ -184,6 +184,9 @@ def compute_deformations():
             # Compute the maximum shear strain rate
             eps_II = sqrt(  (dudx - dvdy)**2 + (dudy + dvdx)**2  )
 
+            # Compute the vorticity
+            eps_III = dvdx - dudy
+
```

Comment:
I do not know if eps_III is an appropriate name for the vorticity; what does everyone think?